subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -125,3 +126,89 @@ def build_lts_payloads() -> dict:
             "$schema": "urn:rhods-matbench-upload:2.0.0",
             **_convert_steps(data)
         }, start_time, end_time
+
+
+def build_limited_lts_payload() -> dict:
+    for(_, entry) in common.Matrix.processed_map.items():
+        RESULTS = entry.results
+        
+        start_time: datetime.datetime = RESULTS.start_time
+        end_time: datetime.datetime = RESULTS.end_time
+
+        output = {
+            "$schema": "urn:rhods-matbench-upload:3.0.0",
+            "data": {
+                "users": _decode_limited_users(RESULTS.ods_ci, RESULTS.testpod_hostnames, RESULTS.notebook_pod_times),
+                'rhods_version': RESULTS.rhods_info.version,
+                'ocp_version': RESULTS.sutest_ocp_version,
+                'metrics': _gather_prom_metrics(RESULTS.metrics),
+                'thresholds': RESULTS.thresholds,
+                'config': RESULTS.test_config.yaml_file
+            },
+            "metadata": {
+                "test": "rhods-notebooks-ux",
+                "start": start_time.isoformat(),
+                "end": end_time.isoformat()
+            }
```

Comment:
I don't see the settings here or in the `data`?

Reply:
yes, but I'd like to see the `entry.settings` dictionary, that's what I want to use to query/filter the results from the server